### DISCUSSION TITLE
Confusion on how to make templ files have completion and intelisense in vscode

### DISCUSSION THREAD

**Discussion by Captain-Leftovers** [a-h/templ#523] [General]
Is there a specific way to make templ files to to have html completeion 
the file does not recognise any html attributes and its really hard do reason "because of my skill level" what exactly is the issue can someone that knows if i can even have html completion and lets say tailwind intelisense in the templ files in vs code please help me how to set it up

**Comment by a-h** [comment]
Not sure if you saw the docs about Tailwind etc at https://templ.guide/commands-and-tools/ide-support

There's discussion about HTML completion more generally at https://github.com/a-h/templ/issues/498

**Reply by Captain-Leftovers** [reply]
Thank you
Yes i saw the discussion for HTML i think it will be really beneficial
At the moment i am switching between HTML and templ file assossiation because i cant have both in vscode 
Thank you for the information and for TEMPL
